homeassistant/components/overkiz/climate_entities/somfy_thermostat.py

```diff
@@ -0,0 +1,167 @@
+"""Support for Somfy Smart Thermostat."""
+from __future__ import annotations
+
+from typing import Any, cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    PRESET_AWAY,
+    PRESET_HOME,
+    PRESET_NONE,
+    ClimateEntityFeature,
+    HVACAction,
+    HVACMode,
+)
+from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+
+from ..coordinator import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+PRESET_FREEZE = "freeze"
+PRESET_NIGHT = "night"
+
+STATE_DEROGATION_ACTIVE = "active"
```

Comment:
A very minor comment: Is `DEROGATION` directly translated from French? Maybe consider trying to find a better translation.

Reply:
We named it like this because it’s one of the value of [CORE_DEROGATION_ACTIVATION](https://github.com/home-assistant/core/pull/67169/files/3fabfd4c3d52bfabf070834d8f2971b2a9fea210#diff-7b52214972019e7783fcb8b4528bc473433cd1f1d6476d57c2203f6d3844581bR77) which is the name used by Somfy in their API